clippy_utils/src/higher.rs

```diff
@@ -502,6 +521,8 @@ impl FormatExpn<'tcx> {
 
 /// A parsed `format_args!` expansion
 pub struct FormatArgsExpn<'tcx> {
+    /// The fist argument, the fromat string, as an expr
+    pub format_string: &'tcx Expr<'tcx>,
     /// Span of the first argument, the format string
     pub format_string_span: Span,
```

Comment:
Should I just remove `format_string_span` as the span can be easily be found from the expr